ma_gym/envs/checkers/checkers.py

```diff
@@ -34,6 +35,13 @@ def __init__(self, full_observable=False, step_cost=-0.01, max_steps=100):
         self.full_observable = full_observable
 
         self.action_space = MultiAgentActionSpace([spaces.Discrete(5) for _ in range(self.n_agents)])
+        self._obs_high = np.array([1.0, 1.0] + [max(OBSERVATION_MEANING.keys())] * 9)
```

Comment:
I see that in pong duel , you made the correct change of private (self.__) whereas over here it's protected (self._).
As such, it wouldn't create much of difference for our functionality. However, let's just be consistent with it in all files.

```suggestion
        self.__obs_high = np.array([1.0, 1.0] + [max(OBSERVATION_MEANING.keys())] * 9)
```
same applies for ```self._obs_low```

Reply:
Thanks! Addressed